src/api/pool_api.js

```diff
@@ -427,7 +439,54 @@ module.exports = {
             auth: {
                 system: 'admin'
             }
-        }
+        },
+
+        get_namespace_resource_operator_info: {
+            doc: 'Return namespace resource operator related info',
+            method: 'GET',
+            params: {
+                type: 'object',
+                required: ['name'],
+                properties: {
+                    name: {
+                        type: 'string',
+                    },
+                }
+            },
+            reply: {
+                type: 'object',
+                properties: {
+                    access_key: { $ref: 'common_api#/definitions/access_key' },
+                    secret_key: { $ref: 'common_api#/definitions/secret_key' },
+                    need_k8s_sync: {
+                        type: 'boolean'
+                    }
+                }
+            },
+            auth: {
+                system: 'admin'
+            }
+        },
+
+        unset_namespace_store_sync: {
```

Comment:
maybe better called sync_namespace_store ? WDYT ?

Reply:
this function puts the namespacestore details on the namespace resource and turns off the sync flag, not sure sync_namespace_store name describes it the best..

Reply:
right, something like set_namespace_store_info ?
Since it does more than just the unset ?

Reply:
sure, updated.